cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1072,6 +1127,154 @@ def cancel(self, req):
+        else:
+            rsp.progress = task.downloadSize * 90 / task.expectedSize
+            rsp.installPath = self.get_direct_upload_path(req[http.REQUEST_HEADER]['Host'])
+
+        if task.lastError is not None:
+            rsp.success = False
+            rsp.error = task.lastError
+        return jsonobject.dumps(rsp)
+
+    @replyerror
+    def copy_file_to_host(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.14.14)

[error] Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)